File: CSIT111/Lecture_2/Lecture_Labs/Lecturer_Examples/TestSwap.py
class TestSwap:
    def main():
        
        m = 123
        n = 456
        
        print("Before the swaps")
        print("\nm = ", m)
        print("\nn = ", n)
        
        # How to swap? Can I do ... no, m and n will have the same values
        m = n
        n = m
        
        print("\nAfter the swaps")
        print("\nm = ", m)
        print("\nn = ", n)
        
testswap = TestSwap()
testswap.main()

File: CSIT111/Lecture_2/Lecture_Labs/Lecturer_Examples/TestSwap.java
// How to swap two values in Java?
// 
// File name: TestSwap.java
class TestSwap
{
	public static void main (String [ ] args)
	{
		int m = 123;
		int n = 456;
		
		System.out.println ("Before the swaps");
		System.out.println ("\tm = " + m);
		System.out.println ("\tn = " + n);
		
		// How to swap? Can I do ... no, m and n will have the same values
		m = n;
		n = m;
		
		System.out.println ("\nAfter the swaps");
		System.out.println ("\tm = " + m);
		System.out.println ("\tn = " + n);
		
	}
}